jupyterhub/apihandlers/users.py

```diff
@@ -209,14 +218,16 @@ def post(self, name):
             raise web.HTTPError(400, "%s's server is not running" % name)
         self.set_server_cookie(user)
         # a service can also ask for a user cookie
-        # this code prevent to raise an error
+        # this code prevents to raise an error
+        # cause service doesn't have 'other_user_cookies'
         if getattr(current, 'other_user_cookies', None) is not None:
             current.other_user_cookies.add(name)
 
 
 default_handlers = [
     (r"/api/users", UserListAPIHandler),
     (r"/api/users/([^/]+)", UserAPIHandler),
-    (r"/api/users/([^/]+)/server", UserServerAPIHandler),
+    (r"/api/users/([^/]+)/servers", UserCreateServerAPIHandler),
```

Comment:
> Sorry, I don't quite know what you are asking here.
Nevermind :)

> never having a server with a name
I would think as "never have to specify a name", so would be /user/{name}/1 by default.
What do you think ?

> only /user/:name URL ever proxied to a single server
what do you mean with this ?

> never allowing a second server to start
Mmmm... this should be add to the config, like enable_multiserver = True or False